bin/run-ted.sh

```diff
@@ -17,4 +18,5 @@ python -u DeepSpeech.py \
   --dropout_rate 0.30 \
   --default_stddev 0.046875 \
   --learning_rate 0.0001 \
+  --checkpoint_dir "$XDG_DATA_HOME/$(basename ${0%.*})" \
```

Comment:
This contains the `run-` prefix which I think is not desirable. Ideally you could do this in one step
```bash
...
--checkpoint_dir "$XDG_DATA_HOME/$(basename -s .sh ${0#run-})" \
...
```
But that doesn't seem to work due to some bash foo I don't have.

So one can do it in two steps.
```bash
XDG_DATA_HOME=...
CHECKPOINT_DIR=$(basename ${0%.*})
CHECKPOINT_DIR=${CHECKPOINT_DIR#run-}
...
   --validation_step 5 \
   --checkpoint_dir "$XDG_DATA_HOME/$CHECKPOINT_DIR" \
...
```